src/Compilers/Core/Portable/Emit/SemanticEdit.cs

```diff
@@ -154,11 +178,17 @@ public SemanticEdit(SemanticEditKind kind, ISymbol? oldSymbol, ISymbol? newSymbo
             Kind = kind;
             OldSymbol = oldSymbol;
             NewSymbol = newSymbol;
-            PreserveLocalVariables = preserveLocalVariables;
             SyntaxMap = syntaxMap;
             Instrumentation = instrumentation;
         }
 
+        /// <summary>
+        /// True if <see cref="SyntaxMap"/> is not null.
+        /// </summary>
+        [Obsolete("Use SyntaxMap instead")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why hide this property and mark as obsolete?

Reply:
I guess we can keep it.